Changelog.md

```diff
@@ -2,6 +2,14 @@
 
 ### Ignition CMake 2.X.X (20XX-XX-XX)
 
+1. Make the OGRE plugin path discovery portable
+  * [Pull request #182](https://github.com/ignitionrobotics/ign-cmake/pull/182)
```

Comment:
I don't think we usually link the forward port PR in the changelog - just the PR that made the change, right?

Reply:
Yeah that's what we've been doing... And we add the changelog at the moment of release so it's easier to generate it. Are you ok with that @j-rivero ?